app/src/main/res/values-cs/strings.xml

```diff
@@ -0,0 +1,441 @@
+  <string name="receive_default_desc">Phoenix platba</string>
+  <string name="receive_generate_button">Vytvořit požadavek</string>
+
+  <string name="receive_share_subject">Platba vyžádána</string>
+  <string name="receive_share_title">Sdílet tento požadavek pomocí…</string>
+  <string name="receive_share_button">Sdílet…</string>
+
+  <string name="receive_address_label">Adresa</string>
+  <string name="receive_power_save_disclaimer">Jste v módu šetření baterie. Pro přijetí platby se ujistěte, že aplikace může běžet na pozadí.</string>
+  <string name="receive_withdraw_button">Použít LNURL odkaz</string>
+
+  <string name="receive_swap_in_button">Zobrazit Bitcoinovou adresu</string>
+  <string name="receive_swap_in_error_message">Nastala chyba při směně na on-chain transakci.…\n\nProsím, zkuste to znovu.</string>
+  <string name="receive_swap_in_progress_message">Požaduji směnu na bitcoinovou adresu…</string>
+  <string name="receive_swap_in_disclaimer_title">Přijmout on-chain</string>
+  <string name="receive_swap_in_disclaimer_message"><![CDATA[
+  Bude zobrazena standardní bitcoinová adresa.
+  <br /><br />
+  Finance poslané na tuto adresu se zobrazí po jednom potvrzení.
+  <br /><br />
+  Poplatek <b>%1$s%%</b> bude aplikován.
+  ]]></string>
+
+  <!-- //////////////// receive with open channel //////////////// -->
+
+  <string name="receive_with_open_title">Příchozí platba</string>
+  <string name="receive_with_open_help_desc">Co to je?</string>
+  <string name="receive_with_open_help">Pro přijetí Lightning platby musíte mít dostatečnou příchozí likviditu.\n\nPokud žádnou nemáte, Phoenix automaticky vytvoří kanál dle potřeby. Zřizovací poplatek pokryje cenu vytvoření kanálu a přidání likvidity do něj.</string>
+  <string name="receive_with_open_amount_label">Máte nevyřízenou příchozí platbu</string>
+  <string name="receive_with_open_cost"><![CDATA[Přijmutí této platby vyžaduje sestavovací poplatek <b>%1$s</b> (%2$s).]]></string>
+  <string name="receive_with_open_accept">Přijmout (vyprší za %1$ss)</string>
+  <string name="receive_with_open_decline">Odmítnout</string>
+  <string name="receive_with_open_expired">Tato platba vypršela!</string>
+
+  <!-- //////////////// send //////////////// -->
+
+  <string name="send_init_paste">Vložit platební požadavek</string>
+  <string name="send_init_reading">Čtu sken…</string>
+
+  <string name="send_balance_prefix">Zůstatek:</string>
+  <string name="send_use_all_balance">Poslat vše</string>
+  <string name="send_loading">Kontroluji platbu…</string>
+  <string name="send_invalid_pr_generic">Tato platba není platná. Musel jste naskenovat neplatný požadavek.\n\nProsím zkuste to znovu.</string>
+  <string name="send_amount_error">Prosím, vložte platné množství.</string>
+  <string name="send_amount_error_balance">Množství je vyšší než váš zůstatek.</string>
+  <string name="send_amount_error_swap_out_too_small">Minimálně 10000 satosthi za on-chain platbu.</string>
```

Comment:
Misspelled "satoshi"

I would leave it as "sat"

Reply:
Are we sure that users recognize "sat" as "satoshi" well enough?

Reply:
My guess is that both sat and satoshi are not recognized well enough. A user which doesn't know the term sat will most likely google the term satoshi.

Reply:
Still not sure about this one. Would rather leave it as "satoshi".